src/styles/header.module.scss

```diff
@@ -0,0 +1,19 @@
+.header-text {
```

Comment:
Instead of pixel units, it's better for responsiveness and accessibility to use rem-units as they scale based on the browsers base font size. 
In addition all of these should be converted to variables (see `_variables.scss`) to make the code more consistent and easier to maintain in the long run.

And sorry for nitpicking so much 🙏  But they could also be ordered alphabetically. There are different opinions about which is better, but ordering them based on the alphabet generally makes it easy to find the properties later and the other files are ordered like that already 😁